hostscripts/rpm-packaging/createproject.py

```diff
@@ -59,10 +59,13 @@ def upload_meta(project, build_repository, linkproject):
     description = ''
     if linkproject:
         if 'ZUUL_UUID' in os.environ:
-            description = """
+            description += """
 This project tests the following Zuul Change IDs: %(ZUUL_CHANGE_IDS)s\n
 Branch used: %(ZUUL_BRANCH)s\n
 Project used: %(ZUUL_PROJECT)s
+""" % (os.environ)
+        if 'BUILD_URL' in os.environ:
+            description += """This project was triggered by %(BUILD_URL)s
```

Comment:
IMO we should try to put this into the '''<url>''' tag so that it is a clickable url instead.